compiler/src/dotty/tools/dotc/transform/Splicer.scala

```diff
@@ -172,6 +158,15 @@ object Splicer {
           ex.printStackTrace(new PrintWriter(sw))
           sw.write("\n")
           throw new StopInterpretation(sw.toString, pos)
+        case ex: InvocationTargetException =>
+          val sw = new StringWriter()
```

Comment:
Why use a low-level method to compose the output? Is it to avoid a stackoverflow?

Reply:
it was just to pass it simply to the  `ex. printStackTrace` method of `Exception`.

Reply:
Later I will need to improve the format of the output, then we will switch to a high level abstraction.